ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -76,3 +76,15 @@ if [ -n "${FEATURE_GATES}" ]; then
 featureGates: ${FEATURE_GATES}
 EOF
 fi
+
+if [ ${ADDITIONAL_NTP_SERVERS,,} = "true" ]; then
```

Comment:
IMO, these "additional" NTP servers - in the RDU2 network - should be passed by any non-dhcp-based configuration. Should we rather set this if statement to work with any job >=4.18 when no dhcp is used to save this variable?

Reply:
This feature is not planned to be tested functionally in any of the job profiles,  except for the dedicate job `baremetal-ipi-ovn-ipv4-vmedia-disruptive-arm/amd` we have created for highly disruptive tests.